src/_layouts/docs.html

```diff
@@ -51,22 +51,26 @@ <h3 class="sidebar-sectionTitle">{{ section.title | escape }}</h3>
 
   </nav>
   <article class="docsContent">
-    {% if page.hideEditButton != true %}
-    <a class="editOnGitHub u-hideOnMobile" 
-       href="https://github.com/tilt-dev/tilt.build/tree/master/docs/{{page.name}}">
-      {% svg assets/svg/github.svg width=16 height=16 %}
-
-      <span class="editOnGitHub-text">
-        Edit on GitHub
-      </span>
-    </a>
-    {% endif %}
-    
     <h1>{{ page.title | escape }}</h1>
     
     {{ content }}
 
+    <div class="docsFooter">
     {% include helpful_form.html %}
+    <a id="scrollToTop" href="#">
+      <span aria-hidden="true">↑</span>
+      Back to top
+    </a>
+    {% if page.hideEditButton != true %}
+      <a class="editOnGitHub u-hideOnMobile"
+         href="https://github.com/tilt-dev/tilt.build/tree/master/docs/{{page.name}}">
+        {% svg assets/svg/github.svg width=16 height=16 %}
```

Comment:
(tiny nitpick) maybe worth adding the same semantic markup to this svg asset too? or you could add an `alt="github logo"`.

Reply:
WHY
NOT
BOTH

(6c98e1f)

Reply:
hahaha well i wonder if they kind of cancel each other out. like the alt text is there for assistive tech to read, but then you're saying that that element doesn't have any meaning. (or at least, that's what i thought/think. the formal docs on [presentation](https://www.w3.org/TR/using-aria/#presentation) are more complicated than i realized.) tldr; alt text should be good enough!